### ISSUE TITLE
CraftTweaker Recipe for Witch Eye broken

### ISSUE THREAD

**Issue by MercuryDoll** [chocolate-edition/Chocolate-Edition#40]
The crafting recipe for the witch eye is impossible, as two of the potions used in it are "Uncraftable Potions", which _are_ craftable, but the eye recipe actually calls for two different potions (as denoted by the NBT tags, which are important). I assume this was overlooked when removing Tough As Nails from the modpack, as the second potion references that mod.

**Comment by RichPrune** [comment]
Update to 1.5.3.2

**Comment by MercuryDoll** [comment]
Just did, the CT script `witch_eye.zs`  remains unchanged and the eye remains uncraftable by legitimate means.

**Comment by GildedToxin** [comment]
Are you playing single player? Or multiplayer?

**Comment by GildedToxin** [comment]
Just pulled up the scripts and it's definitely still there with the tough as nails potion

**Comment by MercuryDoll** [comment]
Local server with friends, but this would mostlikely be a singleplayer issue as well.

**Comment by GildedToxin** [comment]
Yeah, it definitely is the script being wrong, it will be fixed in the next update, if you don't wanna wait till then you can /give it to yourself
